examples/esp-js-react-agile-board/tslint.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
tslint walks the tree, so you can move this right to the root and all projects will use it.

Reply:
This is the only place that has typescript ATM